babel/messages/pofile.py

```diff
@@ -18,6 +18,8 @@
 from babel.util import wraptext
 from babel._compat import text_type
 
+__all__ = ['PoFileError']
```

Comment:
Remove this, please. We haven't had an `__all__` stanza in this file, so adding one that just exports a single name will break `from babel.messages.pofile import *` imports in client code.

Reply:
👍